docs/dev-guide/deployments_table_migration.md

```diff
@@ -0,0 +1,85 @@
+# Deployments Table Migration
+
+This doc outline the data migration steps required for [PR #483 - Refactor deployments storage on redeploy and undeployment](https://github.com/caraml-dev/merlin/pull/483).
+
+## Migration Scripts
+
+There's two distinct scenario of deployments data to be migrated:
+
+1. For version endpoints in terminated status, and
+2. For version endpoints in pending/running/serving status
+
+### For terminated version endpoints
+
+If endpoint status = terminated, update all succesful deployments status to terminated
+
+```
+UPDATE
```

Comment:
Why was the `db-migrations/33_update_multiple_deployments.up.sql` file removed? Can we add it back and reference it here in the doc? It's ok that the `.down.sql` file is empty. Having the migration file will make it easy to roll out the migration to all environments as well.

Reply:
It's not removed as I can see it in the latest commit of this PR's branch (`redeploy-update-deployments-table`): https://github.com/caraml-dev/merlin/blob/redeploy-update-deployments-table/db-migrations/33_update_multiple_deployments.up.sql

Seems like you were looking at previous commit.

<img width="500" alt="image" src="https://github.com/caraml-dev/merlin/assets/8122852/7795d179-5b42-47f3-a99d-a2e8355765fe">